services/api/routers/crud.py

```diff
@@ -0,0 +1,58 @@
+"""crud functions"""
```

Comment:
maybe this file should go in a different directory to keep router stuff separate?